stackgl_modules/index.js

```diff
@@ -1,43 +1,33 @@
-(function webpackUniversalModuleDefinition(root, factory) {
```

Comment:
The `umd` header is not required. 
It is now simplified to a `commonjs` module.